src/helper/Reply.h

```diff
@@ -63,6 +63,10 @@ class Reply
 
     /// The stateful replies should include the reservation ID
     Helper::ReservationId reservationId;
+
+    /// Time after which this response should no longer be used
+    time_t expires = 0;
```

Comment:
I recommend using std::optional<time_t> type to improve code quality, including self-documentation and removal of `if` statements. If you insist on using zero as a special value, then please document that special value by adding the corresponding line to this data member description.

Reply:
Done